mito-ai/mito_ai/utils/message_history_utils.py

```diff
@@ -33,7 +34,8 @@ def trim_sections_from_message_content(content: str) -> str:
         GET_CELL_OUTPUT_TOOL_RESPONSE_SECTION_HEADING,
         ACTIVE_CELL_OUTPUT_SECTION_HEADING,
         ACTIVE_CELL_ID_SECTION_HEADING,
-        STREAMLIT_APP_STATUS_SECTION_HEADING
+        STREAMLIT_APP_STATUS_SECTION_HEADING,
+        CODE_SECTION_HEADING,
```

Comment:
Currently we have `MESSAGE_HISTORY_TRIM_THRESHOLD=3` which means that the previous active code cell or notebook will only be preserved for 2 user messages. I kinda feel like that is not enough. What if we updated this code slightly so that we trimmed all of the variables at 3, but we preserved the notebook for a few more conversation turns. Sometimes I will say things like "Can you put the code back to where it was at X"